gtest_parallel_tests.py

```diff
@@ -333,6 +342,133 @@ def root():
                                    os.path.join(root(), 'c', 'd', 'bin'),
                                    'Test.case', 1))
 
+    log_file = gtest_parallel.Task._logname(None, None, None, None)
```

Comment:
Can you make this a separate case? Just `def test_logs_to_temporary_files_without_output_dir:` ?

Reply:
Can't comment on the right line (because GitHub), but line 319, isn't this wrong? If we log to empty string I think that should be the same as None, I think `--output_dir=` should probably be rejected or equivalent to omitting `--output_dir`. I think `--output_dir=.` should be used for the behavior on that line. (This is the `_logname('',` call.)

Reply:
> line 319, isn't this wrong?

I don't see anything wrong with it. However, it can never happen on a normal run. Even if `--output_dir` is `''` on the command line it changes to `gtest-parallel-logs` in `main()`

Reply:
If it can't ever happen I think `'.'` is a better test case as we can definitely provide that argument.